{"name": "Проверка получения ошибки при авторизации курьера с неправильным паролем", "status": "passed", "start": 1723111275467, "stop": 1723111275703, "uuid": "838ffe59-cb5d-47ce-b9df-1860d0bb71e7", "historyId": "a2eadefbbbef6f5be40567c1f4abdf55", "testCaseId": "a2eadefbbbef6f5be40567c1f4abdf55", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_with_wrong_pass_shows_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "Nightingales-MacBook-Air.local"}, {"name": "thread", "value": "2738-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}